blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_zh.properties

```diff
@@ -0,0 +1,286 @@
+# common
+common.date.duration.format=d[d] h[h] m[m] s[s]
+common.date.duration.hint.format=M [month], d [days], h[ hours], m[ minutes], s[ seconds]
+common.date.duration.display.format=d[d] h[h] m[m] s[s]
+common.date.readable.long=MMM DD YYYY h:mma Z
+common.date.readable.short=MMM DD h:mma Z
+#加载中...
+common.pager.loading=\u52a0\u8f7d\u4e2d\u002e\u002e\u002e
```

Comment:
It must be unicode，otherwise it will like this picture.
“#...” just convenience to next modify

![2017-07-27 15 19 52](https://user-images.githubusercontent.com/4201186/28658774-6a8780fc-72df-11e7-86c0-f1b024912f9e.png)